src/content-steering-controller.js

```diff
@@ -77,13 +77,14 @@ export default class ContentSteeringController extends videojs.EventTarget {
     this.defaultPathway = null;
     this.queryBeforeStart = null;
     this.availablePathways_ = new Set();
-    // TODO: Implement exclusion.
     this.excludedPathways_ = new Set();
     this.steeringManifest = new SteeringManifest();
     this.proxyServerUrl_ = null;
     this.manifestType_ = null;
     this.ttlTimeout_ = null;
     this.request_ = null;
+    this.requestError_ = null;
+    this.excludedSteeringManifestURLs = [];
```

Comment:
Based on my understanding of the code, it is better to use `Set` instead of `Array`

Reply:
`Set` uses hashes which makes the `includes` operation constant instead of O(n).